src/core.e/control.pm6

```diff
@@ -1,4 +1,7 @@
 multi sub next(\x --> Nil) { THROW(nqp::const::CONTROL_NEXT, x) }
 multi sub last(\x --> Nil) { THROW(nqp::const::CONTROL_LAST, x) }
 
+proto sub prefix:<//>($) {*}
+multi sub prefix:<//>(\a) { a.defined }
```

Comment:
Probably this wants to be marked `is PURE`. That way, some syntax accidents can be detected in the case where the language user thought they were writing an infix `//`, but it's actually a useless prefix `//` in sink context. One particular case that comes to mind is:

```
try { foo() }
    // say "oops";
```

The `}` at line end is a statement terminator, so `//` cannot parse as an infix here. Today there's a (not terribly awesome) syntax error raised mentioning the null regex, so such a mistake won't make it past compilation (although might take an effort to understand what happened). Marking the `proto` here with `is PURE` should at least result in a potential difficulty being reported, although those are easier to ignore.

Reply:
So done with https://github.com/rakudo/rakudo/pull/5040/commits/e3abea7d4dff3faf8a0a594929c1bed1684c3789